docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -55,23 +55,27 @@ outside of those test subdirectories.  They will also not reconfigure or bounce
 
 Run as follows for all POSIX platform tests executed by our CI system::
 
-    test/runner/ansible-test integration --docker fedora25 -v posix/ci/
+    ansible-test integration --docker fedora29 -v shippable/
```

Comment:
Sorry, I should have been more clear on my previous feedback. You'll probably want to add a brief section mentioning use of `source hacking/env-setup` (as is found in several other places in our docs, although not currently in this file), so that `ansible-test` is in the path. Then update this example, as you've done, along with all the others in this file to use `ansible-test` instead of `test/runner/ansible-test`.

Reply:
Is it better this way: https://github.com/ansible/ansible/pull/53338/commits/205b960a662e6fccb9747605cfdf3a3d483c97b3

As way tempted to just add `source hacking/env-setup` before every ansible-test example. This may be enough.